src/Streamly/Internal/Data/Stream/IsStream/Generate.hs

```diff
@@ -574,16 +582,20 @@ fromFoldable :: (IsStream t, Foldable f) => f a -> t m a
 fromFoldable = fromStream . K.fromFoldable
 
 -- |
--- @
--- fromFoldableM = 'Prelude.foldr' 'consM' 'K.nil'
--- @
+-- >>> fromFoldableM = Prelude.foldr Stream.consM Stream.nil
 --
 -- Construct a stream from a 'Foldable' containing monadic actions.
 --
--- @
--- drain $ fromSerial $ S.fromFoldableM $ replicateM 10 (threadDelay 1000000 >> print 1)
--- drain $ fromAsync  $ S.fromFoldableM $ replicateM 10 (threadDelay 1000000 >> print 1)
--- @
+-- >>> pr n = threadDelay 1000000 >> print n
```

Comment:
"pr n = threadDelay 1000000 >> print n" is defined in multiple places can be defined one place.

Reply:
The problem is that if we define it at one place then the name has to be intuitive to know what it is doing. By redefining readers know immediately from the definition. I think we have defined it as "delay" in other modules but the problem is that we have another "delay" in prelude.